src/neuroconv/datainterfaces/ophys/miniscope/miniscopeconverter.py

```diff
@@ -0,0 +1,77 @@
+from typing import Literal
+
+from pynwb import NWBFile
+
+from ... import MiniscopeBehaviorInterface, MiniscopeImagingInterface
+from ....nwbconverter import NWBConverter
+from ....utils import FolderPathType, get_schema_from_method_signature
+
+
+class MiniscopeConverter(NWBConverter):
+    """Primary conversion class for handling Miniscope data streams."""
+
+    @classmethod
+    def get_source_schema(cls):
+        return get_schema_from_method_signature(cls)
+
+    @classmethod
+    def get_conversion_options_schema(cls):
+        return get_schema_from_method_signature(
+            MiniscopeImagingInterface.add_to_nwbfile, exclude=["nwbfile", "metadata"]
+        )
```

Comment:
Might actually just be able to effectively alias this

```suggestion
        return MiniscopeImagingInterface.get_conversion_options_schema()
```

Reply:
Since the other interface doesn't have anything

Reply:
This returns: 
```
>       return MiniscopeImagingInterface.get_conversion_options_schema()
E       TypeError: get_conversion_options_schema() missing 1 required positional argument: 'self'
```
so maybe still `get_schema_from_method_signature` is needed?